main.go

```diff
@@ -73,3 +95,31 @@ func albumsByArtist(name string) ([]Album, error) {
   return albums, nil
 }
 
+// albumById queries for the album with the specified ID.
+func albumById(id int64) (Album, error) {
+  // An album to hold data from the returned row.
+  var album Album
+
+  row := db.QueryRow("SELECT * FROM album WHERE id = ?", id)
+  if err := row.Scan(&album.ID, &album.Title, &album.Artist, &album.Price); err != nil {
+    if err == sql.ErrNoRows {
```

Comment:
Careful with `err ==`!
Remember to use `errors.Is` when you want to check if an error matches some other error, so that wrapped errors are still counted as `true`.

Reply:
FIXED